src/utils/path_utils.ts

```diff
@@ -38,6 +38,10 @@ export class pathUtils {
         return repoPath.replace(":", "");
     };
 
+    getFileRelativePath  = (tabPath: string): string => {
```

Comment:
Is this going to be used at somewhere else? If not, probably no need to make a function of it

Reply:
Not that I can think of. I made it just in case we needed it in future. I'll remove it then